### PR TITLE
maxOfVector works

### PR DESCRIPTION
**Pull request by pwiatr3**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: max-of-vector
🏅 Granted 5 XP for @pwiatr3

**Comment by ziobron** [comment]
⏱ 3 XP extra for @pwiatr3 for on-time delivery

File: homework/max-of-vector/maxOfVector.hpp [hunk 1]
```diff
@@ -4,5 +4,12 @@
 
 int maxOfVector(const std::vector<int>& vec) {
     // TODO: Implement me :)
-    return {};
+    int max = vec[0];
```
Comment (nauka-programowania-MA) [inline homework/max-of-vector/maxOfVector.hpp:7]: Vector could be empty. Also mark result as const.

File: homework/max-of-vector/maxOfVector.hpp [hunk 2]
```diff
@@ -4,5 +4,12 @@
...
+    for(auto& number : vec){
```
Comment (nauka-programowania-MA) [inline homework/max-of-vector/maxOfVector.hpp:8]: const auto&

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**